tests/conftest.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from blockbuster import blockbuster_ctx
+
+
+@pytest.fixture(autouse=True)
+def blockbuster(request):
+    with blockbuster_ctx("fastapi") as bb:
+        bb.functions["io.BufferedReader.read"].can_block_in("starlette/testclient.py", "receive")
+        for func in [
+            "os.stat",
+            "io.TextIOWrapper.read",
+        ]:
+            bb.functions[func].can_block_in("pydantic/networks.py", "import_email_validator")
+        for version in ["001", "002"]:
```

Comment:
The tutorials do blocking calls.
So I've put some exemptions.
Would you prefer to fix the tutorials ?